### ISSUE TITLE
Address subscription causes large backlog during catch-up sync.

### ISSUE THREAD

**Issue by evoskuil** [libbitcoin/libbitcoin-server#332]
Address subscription, enabled via the `server.subscription_limit` configuration setting, should not be enabled until the node is fully synced. This should be automatically configured internally.

```ini
[server]
# The number of query worker threads per endpoint, defaults to 1 (0 disables service).
query_workers = 1
# The maximum number of subscriptions, defaults to 0 (disabled).
subscription_limit = 10000
```

**Comment by evoskuil** [comment]
Mitigated by eliminating tx copies, but still may be an issue with large blocks.

**Comment by evoskuil** [comment]
See also: https://github.com/libbitcoin/libbitcoin-server/issues/347

**Comment by evoskuil** [comment]
This is resolved for v3.1.
